src/structures/buffers/publickeyrsa.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use crate::tss2_esys::TPM2B_PUBLIC_KEY_RSA;
+use crate::{Error, Result, WrapperErrorKind};
+use log::error;
+use std::convert::TryFrom;
+/// Struct holding a data value.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct PublicKeyRSA {
+    value: Vec<u8>,
+}
+
+impl PublicKeyRSA {
+    const MAX_SIZE: usize = 512;
+    pub fn value(&self) -> &[u8] {
+        &self.value
+    }
+}
+
+impl Default for PublicKeyRSA {
+    fn default() -> Self {
+        PublicKeyRSA {
+            value: Vec::<u8>::new(),
+        }
+    }
+}
```

Comment:
I think just deriving `Default` on `PublicKeyRSA` gives you this as well!

Reply:
Right. The `publickeyrsa.rs` file started as a copy of `data.rs`, as you might've noticed from the other comment.
I think we may want to abstract the buffer types, just as we had an issue for the List types.

Reply:
Yeah, I'll try and think of a way, although Rust might make it somewhat complicated.